tslearn/nonmyopic.py

```diff
@@ -0,0 +1,450 @@
+                                                                                               Y_train2_hat,
+                                                                                               labels=classes_y))
+                    column_sum = self.clusters_["ck_cm{0}".format(current_cluster)][-1].sum(1)
+                    for current_class in range(0, self.number_classes_):
+                        if column_sum[current_class] == 0:
+                            column_sum[current_class] = 1
+                    matrix_proba_current_cluster_class = \
+                        np.asarray([[self.clusters_["ck_cm{0}".format(current_cluster)][-1][0][0] / column_sum[0],
+                                     self.clusters_["ck_cm{0}".format(current_cluster)][-1][0][1] / column_sum[0]],
+                                    [self.clusters_["ck_cm{0}".format(current_cluster)][-1][1][0] / column_sum[1],
+                                     self.clusters_["ck_cm{0}".format(current_cluster)][-1][1][1] / column_sum[1]]])
+                    self.pyhatyck_["pyhatycks{0}".format(current_cluster)].append(matrix_proba_current_cluster_class)
+
+    def get_avg_dist(self, truncated_xt):
+        """
+        Compute the average euclidean distance of a given time series to the centroid of each cluster of the model
+
+        Parameters
+        --------------------
+        xt: Vector
+            A time series that may  be incomplete
+
+        Returns
+        -------------------
+        Float : the average euclidean distance of the time series to the centroid of each cluster of the model
+        Float : the minimum distance between the centroid of a cluster and the time series
+        """
+
+        sum_distances = 0
+        minimum_distance = 0
+        for current_cluster in range(0, self.number_cluster):
+            euclidean_distance = np.linalg.norm(truncated_xt - self.cluster_.cluster_centers_[current_cluster, :len(truncated_xt)])
+            sum_distances = sum_distances + euclidean_distance
+            if minimum_distance < euclidean_distance:
+                minimum_distance = euclidean_distance
+        return sum_distances / self.number_cluster, minimum_distance
+
+    def get_dist(self, truncated_xt, wanted_cluster):
+        """
+        Compute the euclidean distance between a time series xt and the centroid of a cluster of the model
+        Parameters
+        ----------------
+        xt: vector
+            a time series that may be incomplete
+        wanted_cluster: int
+            the number of the cluster for which the distance is wanted
+
+        Returns
+        ----------------
+        Float: the expected euclidean distance
+        """
+
+        euclidean_dist = np.linalg.norm(truncated_xt - self.cluster_.cluster_centers_[wanted_cluster, :len(truncated_xt)])
+
+        return euclidean_dist
+
+    def get_cluster_prob(self, truncated_xt):
```

Comment:
```suggestion
    def get_cluster_proba(self, truncated_xt):
```
to be consistent with scikit-learn's 'predict_proba'